app/org/sagebionetworks/bridge/services/SendMailViaAmazonService.java

```diff
@@ -162,4 +201,53 @@ private String createSignedDocument(ConsentSignature consent, StudyConsent study
         html = html.replace("@@signing.date@@", signingDate);
         return html;
     }
+    
+    private String createInlineParticipantRoster(List<StudyParticipant> participants) {
```

Comment:
It looks like there's some non-trivial logic here. Should we expose this to unit tests so we can get good branch coverage on this?